### PR TITLE
Saved recording are officially g zipped and pose are played back to the standard endpoint

### PR DESCRIPTION
**Pull request by druiz17**
Saved recording are officially gzipped and pose are played back to the standard endpoint

### CODE REVIEW

**Comment by druiz17** [comment]
Test plan
- run the script inputRecorder.js
- click start recording, record some movement.
- click stop recording and then save the recording
- click load and load the file you recently saved. playback that recording
- check if the correct recording is played back.

- download the file below.
- check if the file loads and plays


[10_Matthew_Bow.gz](https://github.com/highfidelity/hifi/files/1046114/10_Matthew_Bow.gz)

File: libraries/controllers/src/controllers/impl/endpoints/StandardEndpoint.h [hunk 1]
```diff
@@ -12,6 +12,11 @@
 
 #include "../Endpoint.h"
 
+#include <DependencyManager.h>
+
+#include "../../InputRecorder.h"
+#include "../../UserInputMapper.h"
```
Comment (sethalves) [inline libraries/controllers/src/controllers/impl/endpoints/StandardEndpoint.h]: does #include <InputRecorder.h> not work here?

Comment (sethalves) [inline libraries/controllers/src/controllers/impl/endpoints/StandardEndpoint.h]: or maybe #include <controllers/InputRecorder.h> ?

File: libraries/controllers/src/controllers/InputRecorder.cpp [hunk 2]
```diff
@@ -153,7 +165,7 @@ namespace controller {
     QJsonObject InputRecorder::recordDataToJson() {
         QJsonObject data;
         data["frameCount"] = _framesRecorded;
-        data["version"] = "1.0";
+        data["version"] = "0.0";
```
Comment (sethalves) [inline libraries/controllers/src/controllers/InputRecorder.cpp]: For any other reviewers -- this version change is on purpose.  The original file format didn't have a version, and Dante has converted the old files to the new format.  We'll pretend that old format never existed.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/323/HighFidelity-Beta-PR10597-77eb75db95db709c38f93c6759524a81eb5ad2a0.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/323/HighFidelity-Beta-PR10597-77eb75db95db709c38f93c6759524a81eb5ad2a0.dmg

**Comment by CainFoool** [comment]
Testing shortly.

File: libraries/controllers/src/controllers/impl/endpoints/StandardEndpoint.h (hunk 1 above)
Comment (druiz17) [inline libraries/controllers/src/controllers/impl/endpoints/StandardEndpoint.h]: Neiter of them work.

**Comment by CainFoool** [comment]
# Test Passed 

Avatar recording and playback works as expected - file in attachment also plays just fine.

File: libraries/controllers/src/controllers/impl/endpoints/StandardEndpoint.h (hunk 1 above)
Comment (jherico) [inline libraries/controllers/src/controllers/impl/endpoints/StandardEndpoint.h]: It's a relative file for an implementation detail, so it's reasonable to use a relative path like this.

**Pull request merged**